From a9010d53ff6fda665dc5ad604ce1b46b872c9747 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 22 May 2018 11:08:48 +0800
Subject: [PATCH] arm64: dts: rockchip: with GPT for rk3326-linux bootargs

The linux platform used the GPT for update image, and the new
tools(v2.55) had fixed the bug before.

Change-Id: I200d98170d538098c4e3472a22398a4ecc15270e
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
index bc353e69bd77..c8b12908b1e7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
@@ -44,7 +44,7 @@
 	compatible = "rockchip,linux", "rockchip,rk3326";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 root=/dev/mmcblk1p8 rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootwait";
 	};
 
 	fiq-debugger {
-- 
2.35.3

